<?xml version="1.0"?>

<launch>
	<node respawn="true" pkg="rovitis_phidgets" type="phidgets_motor_control_hc" name="phidgets_motor_control_hc" output="screen">
		<param name="dev_serial_number" type="int" value="147399" />
		<param name="dev_serial_label" type="string" value="phidgets_motor_control_hc" />
		<param name="frame" type="string" value="/phidget_mc" />
		<param name="wait_for_dev_delay" type="int" value="10000" />
		<param name="publish_frequency" type="int" value="5" />
		<param name="topic_name" type="string" value="/phidgets_motor_control_hc_params" />	
		<param name="vel_cmd_topic_name" type="string" value="/cmd_vel" />
	</node>
</launch>
